.node-version

```diff
@@ -0,0 +1 @@
+12.22.12
```

Comment:
I think this went fine on staging, but I'm noting that Netlify has always built with `v10.24.1` before, so there is a small chance things will turn out slightly differently. So, this is something to look at in the unlikely event that something gets weird on prod.

Though now that I've looked at package.json, this is probably a smaller change than upgrading Svelte.